### ISSUE TITLE
Datepicker "Days" row is not set as table header

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4634]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value 

**Test Environment:**
- Test URL: https://clarity.design/documentation/datepicker
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to toggle datepicker button and press it.
3. Open any particular month, move focus on to particular date using Ctrl+alt+arrow keys and observe spoken feedback.

**Expected result:**
When screen reader user switches columns table header should be announced. For date picker days row should be set as table headers, so that particular day is announced to the screen reader user. For e.g. for 5/18/2020 it will say Monday.

Reference Example: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html

**Actual result:**
No table headers defined in the date picker, so screen reader users will find it really difficult to know which day the date falls on.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
